zap/src/main/java/org/parosproxy/paros/security/SslCertificateServiceImpl.java

```diff
@@ -161,7 +162,7 @@ public KeyStore createCertForHost(CertData certData)
                         new X509CertificateHolder(caCert.getEncoded()).getSubject(),
                         BigInteger.valueOf(serial.getAndIncrement()),
                         new Date(System.currentTimeMillis() - Duration.ofDays(30).toMillis()),
-                        new Date(System.currentTimeMillis() + Duration.ofDays(1000).toMillis()),
+                        new Date(System.currentTimeMillis() + Duration.ofDays(825).toMillis()),
```

Comment:
Would it make more sense for this to be 825-30?

Reply:
Yes, it seems a total validity of 825 days is necessary.

Reply:
Tweaked, thanks.